test/TensorFlow/integration.swift

```diff
@@ -228,6 +228,65 @@ public func test_bool_param2(cond: Bool, // expected-warning {{'cond' implicitly
 // CHECK-NEXT: [[PROGRAM:%.*]] = apply [[STARTFN:%.*]](
 // CHECK: cond_br [[BOOLVAL]],
 
+public func test_multiple_ifs(status: Bool // expected-warning {{'status' implicitly copied to the accelerator}}
+) {
+  var a = Tensor<Int32>(0);
+  let b = a;
+  if (status) { // expected-note {{value used here}}
+    a += b;
+  }
+  a += b;
+  if (status) {
+    a += b;
+  }
+  a -= b;
+  print (a)
```

Comment:
use _hostOp(a) so that the generated SIL code is cleaner

Reply:
Thanks!